projects/lights_control/test/test_can_settings.c

```diff
@@ -0,0 +1,49 @@
+#include "can.h"
+#include "event_queue.h"
+#include "interrupt.h"
+#include "log.h"
+#include "status.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+#include "can_setup.h"
+#include "lights_events.h"
+#include "structs.h"
+
+// test for initializing the CAN settings
+
+static CANMessageID s_msg_id = 0x1;
+
+void setup_test(void) {
+  gpio_init();
+  interrupt_init();
+  soft_timer_init();
+  event_queue_init();
+}
+
+void teardown_test(void) {}
+
+void test_can_rx_front(void) {
```

Comment:
It would be nice to name your test in a way that reflects what it does (or tests).